float32 distance
bool isLaneChange
int8 roadOption
